src/GraphQL.Federation/GraphQL.Federation.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp3.0</TargetFrameworks>
+    <Description>Apollo Federation for GraphQL.NET</Description>
+    <PackageTags>GraphQL;Federation</PackageTags>
+    <OutputType>Library</OutputType>
```

Comment:
```suggestion
```

and why `Microsoft.NET.Sdk.Web` ?

Reply:
I need to check [http header](https://www.apollographql.com/docs/federation/metrics/#how-tracing-data-is-exposed-from-a-federated-service) to determine if tracing is enabled.

Reply:
https://github.com/graphql-dotnet/graphql-dotnet/pull/2351#discussion_r586732371